vignettes/install-pksim.Rmd

```diff
@@ -69,17 +69,12 @@ unzip("pksim_minimal.zip", exdir = "PKSim")
 file.remove("pksim_minimal.zip")
 ```
 
-Then, include the following code lines at the begining of the projects scripts:
+Then, include the following code lines at the beginning of the projects scripts:
 
 ```r
 ospsuite::initPKSim(pksimFolderPath = "PKSim")
```

Comment:
This only applies for esqlabs-projects, when minimal PK-Sim is located in the `Code` folder.

Reply:
Well, it can also apply for self-contained project that are not esqlabs project.
this is why it's under the section "# Install and use PK-Sim in a project"

Reply:
Ah yes, I am always cofused when GitHub collapses parts of the code..